packages/admin-ui-codemod/src/__tests__/tag-to-box.test.js

```diff
@@ -0,0 +1,112 @@
+const defineInlineTest = require('jscodeshift/dist/testUtils').defineInlineTest
+const transform = require('../tag-to-box')
+
+describe('modify imports', () => {
+  defineInlineTest(
+    transform,
+    {},
+    'import { tag } from "@vtex/admin-ui"',
+    'import { Box } from "@vtex/admin-ui"',
+    'package single import'
+  )
+
+  defineInlineTest(
+    transform,
+    {},
+    'import { tag, Button, Table } from "@vtex/admin-ui"',
+    'import { Box, Button, Table } from "@vtex/admin-ui"',
+    'package multiple imports'
+  )
+
+  defineInlineTest(
+    transform,
+    {},
+    'import { Tag, Button, Table } from "@vtex/admin-ui"',
+    'import { Tag, Button, Table } from "@vtex/admin-ui"',
```

Comment:
shouldn't it be?
```suggestion
    'import { Box, Button, Table } from "@vtex/admin-ui"',
```

Reply:
Nope. This grants that we don't mess with the [`Tag` component](https://admin-ui.vercel.app/guidelines/components/tag)